### PR TITLE
Add Fortinet Sandbox Disk Usage Check

### PR DESCRIPTION
**Pull request by spenneb**
This pull request adds a disk usage check for the Fortinet Sandbox Appliance

### CODE REVIEW

**Comment by jonaskluger** [comment]
Hello Ralf,

thank you for this pull request. We will have a closer look at your patch and discuss internally on how and if we can implement it into the official code base.

Internal Ref: CMK-3457

File: checks/fortisandbox_disk [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        size_bytes = float(size_bytes / 1024)
+        i = i + 1
+    return "%.2f %s" % (size_bytes, size_name[i])
+
+
+def check_fortisandbox_disk(item, params, info):
+    warn, crit = params["disk_usage"]
+    dusage = float(info[0][0][0])
+    dcap = float(info[0][0][1])
+    dperc = float(dcap / dusage)
```
Comment (jherbel) [inline checks/fortisandbox_disk:49]: Hi Ralf,
I started to look into this PR and I think this line is wrong. To get the usage, you want to compute
`dusage / dcap * 100`,
i.e., used disk space or all available disk space, right? The factor of 100 to get a percentage is also missing.
Best
Jörg

**Comment by jherbel** [comment]
Hi Ralf,

besides my comment above, there is a more general issue: checks which monitor a disk usage should use the function `check_diskstat_dict` from `diskstat.include`. In your case, the input dictionary would simple contain the single key `"utilization"`. Furthermore, you can then set `'group': 'diskstat',` in `check_info`, such that there is no need for a new rulespec.

Please let us know if you are interested in pursuing this further.

Best
Jörg

**Comment by jherbel** [comment]
This PR has been closed due to inactivity.

**Pull request closed without merging**